pkg/ring/kv/consul/client.go

```diff
@@ -35,12 +35,12 @@ var (
 
 // Config to create a ConsulClient
 type Config struct {
-	Host              string
-	ACLToken          string
-	HTTPClientTimeout time.Duration
-	ConsistentReads   bool
-	WatchKeyRateLimit float64 // Zero disables rate limit
-	WatchKeyBurstSize int     // Burst when doing rate-limit, defaults to 1
+	Host              string        `yaml:"host"`
+	ACLToken          string        `yaml:"acl_token"`
```

Comment:
The CLI flag should end with `.acl-token`. Then remember to update `arguments.md`